src/pages/AppRoutes.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+
+import Calculator2 from './Calculator2';
+import Navbar from '../components/Navbar';
+import Home from './Home';
+import Quotes from './Quotes';
+import PageNotFound from './PageNotFound';
+
+const AppRoutes = () => (
+  <>
+    <Navbar />
+    <Switch>
+      <Route exact path="/">
+        <Home />
```

Comment:
- [ ] The Home route or root route should be the last route, kindly implement it under `<pageNotFound>` route

Reply:
I implemented it under the `pageNotFound` but the home page started giving "page not found". So I kept it just above `pageNotFound`. Thank you, @Emmyn5600 :pray: